### PR TITLE
Split Pool Button in two Buttons

### LINKED ISSUE #33: Split up the Add to pool button into 2 buttons
Current:
Screen Shot 2022-02-16 at 09 37 32

New:
[Your Wallet] <-> [Liquidity Pool]

### LINKED ISSUE #40: Disable Add to Pool Button until Token picker finished loading assets
bug:when clicking add to pool button while token picker loading it causes crash

To reproduce click add to pool button while token picker still loading on initial page load

### LINKED ISSUE #61: Missing the status panel in Swap
Should be this panel above the Settings

Screen Shot 2022-02-23 at 17 59 24

### LINKED ISSUE #62: ExchangeIcon is not vertically center in Swap section
Screen Shot 2022-02-24 at 09 02 35

### LINKED ISSUE #67: In Pool section, the purple status box is empty when wallet is not connected
Screen Shot 2022-02-24 at 09 11 49

### LINKED ISSUE #72: In Swap section, remove the 0 default value in TokenPicker
... as placeholder is already displayed as `0.00`

### PR DESCRIPTION
**Pull request by eupston**
## Description
closes #40
closes #33
closes #61
closes #67 
closes #62
closes #72

#### Changes

- Splits Pool button into two
- Updates styling of exchange icon

#### Fixes

- fixed bug crash on initial pool component loading when clicking pool button
- fixes small styling issues around bridge page header and swap infobox 
- fixes exchangeIcon centering swap component
- fixes blank poolsummary box display on initial load

#### Removes

- remove swapiconclass component replace with ExchangeIcon

### CODE REVIEW

File: components/shared/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -137,10 +140,11 @@ const ConnectWalletButton: FC<ConnectWalletButtonProps> = ({
 					"&:hover": {
 						color: "white",
 						backgroundColor: color,
+						opacity: "1",
 					},
 				}}
 				onClick={() => {
-					onClick;
+					onClick();
 				}}
```
Comment (ken-centrality) [inline components/shared/ConnectWalletButton.tsx:148]: this can be collapse to `onClick={onClick}`

File: components/shared/TokenPicker.tsx [hunk 2]
```diff
@@ -286,12 +286,20 @@ const TokenPicker: React.FC<{
...
+											  )
+											: ""
+								  }`
+								: `Withdrawable: ${
+										poolConfig?.userPoolShare
+											? poolConfig?.userPoolShare?.coreAssetBalance?.asString(
+													poolConfig?.coreAsset.decimals
+											  )
+											: ""
+								  }`
```
Comment (ken-centrality) [inline components/shared/TokenPicker.tsx:302]: this group

```
`Withdrawable: ${
	poolConfig?.userPoolShare
		? poolConfig?.userPoolShare?.coreAssetBalance?.asString(
				poolConfig?.[whichAsset === "trade"? 'tradeAsset' : 'coreAsset'].decimals
		  )
		: ""
}`
```

**Review by ken-centrality — CHANGES_REQUESTED** [review]
It's good overall but the swapping action needs to be revised...

If you want to add to the pool, the 2 buttons should be

```
FROM                            TO
Your Account       <->          Liquidity Pool
```

And when you want withdrawal from the pool, the 2 buttons are swapped

```
FROM                            TO
Liquidity Pool     <->          Your Account
```

And the dropdown for each button should have 2 options: 
"Liquidity Pool" and "Your Account", similar to what we are doing to the FROM & TO in Bridge section

File: components/shared/ConnectWalletButton.tsx (hunk 1 above)
Comment (eupston) [reply components/shared/ConnectWalletButton.tsx]: should be fixed now

File: components/shared/TokenPicker.tsx (hunk 2 above)
Comment (eupston) [reply components/shared/TokenPicker.tsx:302]: yeah its pretty gnarly aye lol, I'll condense into a function

**Review by ken-centrality — CHANGES_REQUESTED** [review]
Looks good @eupston, couple of things

- I reckon you can knock down #72 while you are at it, hopefully it's straightforward
- The PoolSwapper...
- [x] 1) Can you have this title update to be either `Add to Pool` or `Withdrawal from Pool`
- [ ] 2) We can remove these labels, don't think we need it
- [x] 3) These dropdown box doesn't need to change the color when switching
Screen Shot 2022-02-24 at 14 01 51

**Comment by ken-centrality** [comment]
sorry, I wasn't very clear, the You Account and Liquidity Pool dropdown colors should be dark color similar to the Bridge. Beside that everything looks great to me! 👍

**Review by aidan-starke — APPROVED** [review]

**Pull request merged**